doc/user_documentation.rst

```diff
@@ -144,6 +144,18 @@ It's even easier for a Fedora Rawhide package::
 because 'fedora' distgit is the default, and we automatically pick the default
 branch.
 
+.. note::
+    Please note that an SRPM is downloaded from the specified DistGit
+    only once per Copr build, regardless of the number of enabled
```

Comment:
I tend to say "DistGit instance", not just DistGit, but?? dunno

Reply:
I'll update

Reply:
s/enabled/chroots you build for/

Reply:
or against

Reply:
Fixed